### ISSUE TITLE
Unrecognized command "set-icon"

### ISSUE THREAD

**Issue by blankey1337** [bamlab/react-native-make#6]
After installing via npm, I am unable to run `react-native set-icon`, I get this error:

```
error Unrecognized command "set-icon".
info Run "react-native --help" to see a list of all available commands.
```

![image](https://user-images.githubusercontent.com/42594751/64491012-ecc62480-d217-11e9-9322-cfaa3333d404.png)

**Comment by HariharanIT** [comment]
> After installing via npm, I am unable to run `react-native set-icon`, I get this error:
> 
> ```
> error Unrecognized command "set-icon".
> info Run "react-native --help" to see a list of all available commands.
> ```
> 
> ![image](https://user-images.githubusercontent.com/42594751/64491012-ecc62480-d217-11e9-9322-cfaa3333d404.png)

+1.. I am too getting the same

**Comment by yleflour** [comment]
Hello, is @bam.tech/react-native-make listed as a dev dependency in your react-native project?
This plugin works as an extension of the **local** react-native-cli. Which means that to run it you have to be in the project folder.

To check if it works, `set-icon` should be listed when running `react-native -h`

**Comment by yleflour** [comment]
Can you copy/paste the output when running `react-native --version`?

**Comment by vieraleonel** [comment]
```react-native --version``` output is:
```
react-native-cli: 2.0.1
react-native: 0.59.10
```

**Comment by efstathiosntonas** [comment]
Bumped in the issue using:
```
react-native-cli: 2.0.1
react-native: 0.61.0-rc.3
react-native-make 1.0.2 
```

```
"devDependencies": {
    "@babel/core": "^7.6.0",
    "@babel/runtime": "^7.6.0",
    "@bam.tech/react-native-make": "^1.0.2",
...
```

**Comment by rogerzklotz** [comment]
Also running into the same issue as @efstathiosntonas 

```
react-native-cli: 2.0.1
react-native: 0.60.5
react-native-make 1.0.2 
```

```
"devDependencies": {
    "@babel/core": "7.6.0",
    "@babel/runtime": "7.6.0",
    "@bam.tech/react-native-make": "1.0.2",
...
```

Any thoughts?

**Comment by yleflour** [comment]
Hello, @efstathiosntonas , I am unable to reproduce the issue:

What I did:

```bash
react-native init TestProject --version 0.61.0-rc.3
cd TestProject
yarn add -D @bam.tech/react-native-make
react-native -h
# Check: The  "make" commands are displayed
```

Are you able to reproduce it on another project?

**Comment by efstathiosntonas** [comment]
I ll check again first thing tomorrow morning and I will post back.

I’m guessing the issue originates because i jumped from 0.60.5 to rc3 via the upgrade tool and some packages might be messed up, I’ll delete node_modules and try again.

**Comment by NicoCevallos** [comment]
Hey, I used
```
react-native init TestProject
cd TestProject
npm i -D @bam.tech/react-native-make
```
and when I run `react-native -h` I don't see any new option.
I'm using:
```
react-native-cli: 2.0.1
react-native: 0.61.0 (also tried with 0.61.0-rc.3)
@bam.tech/react-native-make: 1.0.2
```

Is there a way to run it out of react-native-cli?

**Comment by NicoCevallos** [comment]
OK, I suspected that, but thought that could not be the problem, but yes. I used `yarn` instead `npm` and then I see the available plugins with `-h`.
I hope it helps to find the solution soon.

**Comment by yleflour** [comment]
Just published a fix in 1.0.3 

With Yarn, node_modules folder architecture is flat whereas with NPM it is nested

**Comment by efstathiosntonas** [comment]
Thanks @yleflour for fixing it so quick. Sorry for not posting back.
